composer.json

```diff
@@ -13,6 +13,7 @@
     ],
     "minimum-stability": "dev",
     "require": {
+        "php": "~5.3",
```

Comment:
this should be removed IMO. You are forbidding using the bundle on PHP 7, while it should work (as long as you don't use the String util)